_extras/partner_policy.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "Information for Partners"
+permalink: /partners/
+survey_link: 
+---
+
+
+Welcome to the Software Carpentry and Data Carpentry community.  We are excited to welcome you as a member organization. Our instructor training process is a large part of how we spread the teaching culture of the Carpentries to new organizations. We’re looking forward to meeting your team and bringing them up to speed on how we teach so we can have the best impact on our learners.
+
+Your annual membership gives gives you a number of slots in our instructor training program.  This is a training for people who want to become instructors with the Carpentries (Software Carpentry and Data Carpentry). The workshop is a mix of lectures and hands-on lessons and is very interactive where trainees practice giving short lessons using approaches learned and implement some of the teaching techniques which we will discuss. This is training about pedagogical approach to teaching, not technical training specific to particular lessons. Trainees do not need any particular technical background, and we will not be teaching our workshop curriculum to trainees. This workshop is based on our constantly revised and updated open sourced curriculum for instructor training (https://swcarpentry.github.io/instructor-training/). 
+
+Our trainings are all offered as two day, online events.  You can view our training calendar here (INSERT LINK WHEN WE HAVE ONE) and sign up for a training that meets your team’s needs. A few points as you select dates:
+
+* We recommend that you sign up for an event during the first six months of your partnership year.  This ensures your trainees have time to complete the training program and start teaching workshops during your membership year.
```

Comment:
Could we add that we highly recommend that they also try to ensure that the people they want to train have actually attended a workshop before going into training?